test_settings.py

```diff
@@ -0,0 +1,32 @@
+SECRET_KEY = "psst"
+DEBUG = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": ":memory:",
+        "USER": "",
```

Comment:
I think lines 8-11 can be removed.

Reply:
Addressed in d2d1e57928e4c96f9c245357451a9a6ddcd4720c